app/src/components/market/market_fund.tsx

```diff
@@ -179,6 +184,16 @@ const MarketFundWrapper: React.FC<Props> = (props: Props) => {
 
   const probabilities = balances.map(balance => balance.probability)
 
+  const unlockCollateral = async () => {
+    if (!cpk) {
+      return
+    }
```

Comment:
Unlock also checks for cpk being truthy.

Reply:
I don't understand this comment. Just for the record, this is mainly so that typescript doesn't complain, because if this method is called then `cpk` is almost surely not null.